### PR TITLE
Adding the Setup Video 

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**
<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution.

To ensure that someone gets notified, you can also click on "Request a review" and add
the suggested person.

Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact the HSF training convenors.

You may delete these instructions from your comment.

\- HSF Training
</details>

referencing Issue #33

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 ---
 title: Setup
 ---
+
+  <iframe
+      src="https://indico.cern.ch/event/1058838/attachments/2427696/4157672/HSF%20Matplotlib%20Setup.mp4"
+      title= "HSF Matplotlib Setup"
+      width="100%"
+      height="500"
```
Comment (klieret) [inline setup.md:9]: specifying an absolute height while giving the width in percent looks strange to me. How about something like `style="width: 100%; max-height: 500px"`. Is that closer to what you meant?

**Comment by michmx** [comment]
Sorry, just caught this PR (I need to improve how to filter messages in my mail client!). Any reason why is still open, @klieret ? Or can we merge the video for being displayed in the setup?

**Comment by klieret** [comment]
@michmx Let me just try this out and see if I tweak the CSS

**Comment by klieret** [comment]
Actually looks good as-is :)

**Review by klieret — APPROVED** [review]

**Pull request merged**